x/interchaintxs/types/tx.go

```diff
@@ -10,6 +10,12 @@ import (
 	"github.com/gogo/protobuf/proto"
 )
 
+// const interchainAccountIDLimit = 47
+const interchainAccountIDLimit = 128 -
+	len("icacontroller-") -
+	len("neutron1unyuj8qnmygvzuex3dwmg9yzt9alhvyeat0uu0jedg2wj33efl5qmysp02") - // just a random contract address
```

Comment:
this is weird af, but it looks like there is no length constant for string representation of contract addresses